bin/configs/typescript-consolidated-composed-schemas.yaml

```diff
@@ -3,4 +3,7 @@ outputDir: samples/openapi3/client/petstore/typescript/builds/composed-schemas
 inputSpec: modules/openapi-generator/src/test/resources/3_0/composed-schemas.yaml
 templateDir: modules/openapi-generator/src/main/resources/typescript
 additionalProperties:
+  platform: node
```

Comment:
just curious - what was the reason for changing the platform to `node` here? is it important for the fix?

Reply:
I don't think it makes any difference.